src/main/java/lotto/view/InputView.java

```diff
@@ -15,26 +15,71 @@
     private static final String ENTER = "\n";
     private static final int LOTTO_NUMBER_SIZE = 6;
     private static final String PURCHASE_TICKET_MANAGER_MESSAGE = "구입금액을 입력해 주세요.";
+    private static final String MANUAL_TICKET_MANAGER_MESSAGE = "수동으로 구매할 로또 수를 입력해 주세요.";
+    private static final String MANUAL_TICKET_NUMBER_MANAGER_MESSAGE = "수동으로 구매할 번호를 입력해 주세요.";
     private static final String WINNING_NUMBER_MANAGER_MESSAGE = "지난 주 당첨 번호를 입력해 주세요.";
     private static final String BONUS_BALL_MANAGER_MESSAGE = "보너스 볼을 입력해 주세요.";
     private static final String DELIMITER = ",";
     private static final String INPUT_IS_EMPTY_ERROR_LOG = "공백은 들어올 수 없습니다.";
     private static final String LOTTO_NUMBER_SIZE_ERROR_LOG = "로또는 " + LOTTO_NUMBER_SIZE + "자리여야 합니다.";
 
-    public static Ticket buyTicket() {
-        int money = Integer.parseInt(writePurchaseAmount());
-        return new Ticket(money);
+    public static int getMoney() {
+        try {
+            OutputView.printMessage(PURCHASE_TICKET_MANAGER_MESSAGE);
+            String input = scanner.nextLine();
+            checkEmptyString(input);
+            return Integer.parseInt(input);
+        } catch (IllegalArgumentException e) {
+            e.printStackTrace();
+            return getMoney();
+        }
     }
 
-    private static String writePurchaseAmount() {
+    public static int getManualTicketCount() {
         try {
-            OutputView.printMessage(PURCHASE_TICKET_MANAGER_MESSAGE + ENTER);
+            OutputView.printMessage(ENTER + MANUAL_TICKET_MANAGER_MESSAGE );
             String input = scanner.nextLine();
             checkEmptyString(input);
-            return input;
+            return Integer.parseInt(input);
+        } catch (IllegalArgumentException e) {
+            e.printStackTrace();
+            return getManualTicketCount();
+        }
+    }
+
+    public static List<List<Integer>> getManualLottoNumbers(int manualCount) {
+        try {
+            OutputView.printMessage(ENTER + MANUAL_TICKET_NUMBER_MANAGER_MESSAGE);
+            return makeManualLottoNumbers(manualCount);
+        } catch (IllegalArgumentException e) {
+            e.printStackTrace();
+            return getManualLottoNumbers(manualCount);
+        }
+    }
+
+    private static List<List<Integer>> makeManualLottoNumbers(int manualTicketCount) {
+        List<List<Integer>> lottoNumbers = new ArrayList<>();
+        for (int i = 0; i < manualTicketCount; i++) {
+            lottoNumbers.add(makeManualLttoNumber());
+        }
+        return lottoNumbers;
+    }
+
+    private static List<Integer> makeManualLttoNumber() {
```

Comment:
```suggestion
    private static List<LottoNumber> makeManualLttoNumber() {
```
View에서는 로또 번호 String을 받고, 수동로또번호 생성을 담당하는 객체에서 구매행위객체에게 수동로또 티켓을 전달하는 방법으로 수정해보는 것은 어떨까요?